packages/core/js/AEPCore.js

```diff
@@ -266,4 +266,13 @@ module.exports = {
     RCTAEPCore.setAppGroup(appGroup);
   },
 
+  /**
+   * 
+   * @brief This method requests that each extension resets the identities it owns. Each extension responds to this request uniquely.
+   * 
+   */
+  resetIdentities() {
```

Comment:
might be good to document this API too in https://github.com/adobe/aepsdk-react-native/blob/main/packages/core/README.md

Reply:
Documentation is updated.